src/xpcc/architecture/platform/devices/peripherals.ld.in

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2017, Niklas Hauser
+ *
+ * This file is part of the modm project.
```

Comment:
Replace with xpcc BSD header file. I give permission to relicense this code as BSD.